src/main/java/duke/Storage.java

```diff
@@ -35,48 +34,56 @@ public Storage(String filepath, TaskList taskList) {
      *
      * @throws FileNotFoundException Raise Exception if text file does not exist.
      */
-    public void printFileContents() throws FileNotFoundException {
+    public String printFileContents(TaskList tasks) throws FileNotFoundException {
 
-        // System.out.println(filePath);
         File f = new File(filepath); // create a File for the given file path
-        // System.out.println(f.exists());
         Scanner s = new Scanner(f); // create a Scanner using the File as the source
 
         while (s.hasNext()) {
             String toAdd = s.nextLine();
 
             if (toAdd.charAt(1) == 'T') {
                 String task = toAdd.substring(7);
-                taskList.addTodo(task);
+
+                tasks.addTodo(task);
                 counter += 1;
+                System.out.println(counter);
 
             } else if (toAdd.charAt(1) == 'E') {
                 String task = toAdd.substring(7);
                 String name = task.substring(0, task.indexOf("(") - 1);
```

Comment:
Try to avoid duplicate lines of code. Consider helper functions to parse and extract similar functionalities.